pixivpy3/aapi.py

```diff
@@ -58,18 +70,12 @@ def format_bool(self, bool_value):
     # 返回翻页用参数
     def parse_qs(self, next_url):
         if not next_url: return None
-        if sys.version_info >= (3, 0):
-            from urllib.parse import urlparse, unquote
-            safe_unquote = lambda s: unquote(s)
-        else:
-            from urlparse import urlparse, unquote
-            safe_unquote = lambda s: unquote(s.encode('utf8')).decode('utf8')
 
         result_qs = {}
         query = urlparse(next_url).query
         for kv in query.split('&'):
             # split than unquote() to k,v strings
-            k, v = map(safe_unquote, kv.split('='))
+            k, v = map(unquote, kv.split('='))
```

Comment:
You can use `urllib.parse.parse_qs` instead of unquote it by yourself.

Reply:
Good job, i totally forgot that lib and didn't notice what it implemented